plugins/plugin-webpack/plugin.js

```diff
@@ -361,10 +361,16 @@ module.exports = function plugin(config, args = {}) {
             reject(err);
             return;
           }
+          const info = stats.toJson(extendedConfig.stats);
           if (stats.hasErrors()) {
-            const info = stats.toJson(extendedConfig.stats);
-            console.error(info.warnings.join('\n-----\n'));
-            console.error(info.errors.join('\n-----\n'));
+            console.error('Webpack errors:\n' + info.errors.join('\n-----\n'));
+            reject(Error(`Webpack failed with ${info.errors} error(s).`));
```

Comment:
add a return here to stop execution, after the reject() call.

Reply:
Good catch!